docs/providers/aws/guide/serverless.yml.md

```diff
@@ -124,7 +123,9 @@ provider:
     baz: qux
   tracing:
     apiGateway: true
-    lambda: true # optional, can be true (true equals 'Active'), 'Active' or 'PassThrough'
+    lambda: true # Optional, can be true (true equals 'Active'), 'Active' or 'PassThrough'
+  logs:
+    restApi: true # Optional configuration which specifies if API Gateway logs are used
```

Comment:
Isn't that a breaking change?

Reply:
Good question. Theoretically yes, but the intitial implementation we update here wasn't release just yet. We've merged the CloudFormation based solution (which used the old config) a couple of days ago and after that decided to do this refactoring.

Reply:
I see, so yes in that case it's totally fine. Thanks for clarification